wpilibc/src/main/native/cpp/ADIS16448_IMU.cpp

```diff
@@ -409,6 +409,8 @@ void ADIS16448_IMU::Close() {
 }
 
 ADIS16448_IMU::~ADIS16448_IMU() {
+  delete m_reset_in;
+  delete m_status_led;
```

Comment:
These should be in Close() instead (along with setting them to nullptr after the delete).